liquibase-couchbase/src/main/java/liquibase/ext/couchbase/configuration/CouchbaseLiquibaseConfiguration.java

```diff
@@ -20,7 +20,7 @@ public class CouchbaseLiquibaseConfiguration implements AutoloadedConfigurations
     public static final ConfigurationDefinition<Duration> LOCK_TTL_PROLONGATION;
     public static final ConfigurationDefinition<Duration> TRANSACTION_TIMEOUT;
     public static final ConfigurationDefinition<Duration> MUTATE_IN_TIMEOUT;
-    public static final ConfigurationDefinition<Boolean> IS_REACTIVE_TRANSACTIONS;
+    private static final ConfigurationDefinition<Boolean> IS_REACTIVE_TRANSACTIONS;
```

Comment:
Let's not broke API, in test where you need to test it leave it as you did before, this is usage of Liquibase framework.
It's ok that we cannot unit test it

Reply:
Ok, changed `private` to `public`

Reply:
@romchellis could you please take a review again?